brain_observatory_analysis/dev/data_selection_tools.py

```diff
@@ -145,15 +152,15 @@ def limit_to_last_familiar_second_novel(df):
     if 'second_novel' not in df.columns:
         df = add_second_novel_column(df)
     # drop novel sessions that arent the second one
-    indices = df[(df.experience_level == 'Novel >1') & (df.second_novel == False)].index.values
+    indices = df[(df.experience_level == 'Novel >1') & (df.second_novel == False)].index.values # noqa
```

Comment:
(Just a comment, and venting...)
I think this line and all the others with `== False` expression, it makes flake8 unhappy because it thinks that it must be `is False`, but that errors out most of the cases (One of the things I hate about flake8). I guess this noqa was for that, but maybe it is dealt in the config file? We need to discuss about this at some point.